From f62ca366bd96cd2d73a8c65db5d361548dfa2f1a Mon Sep 17 00:00:00 2001
From: Alexander Bokovoy <ab@samba.org>
Date: Wed, 13 Sep 2017 11:37:34 +0300
Subject: [PATCH] Install dcerpc/__init__.py for all Python environments

Also fix whitespace. We use tabs, not spaces in Python/waf code.
---
 source4/librpc/wscript_build | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/source4/librpc/wscript_build b/source4/librpc/wscript_build
index e341432fa14..a826fbed8ea 100644
--- a/source4/librpc/wscript_build
+++ b/source4/librpc/wscript_build
@@ -406,10 +406,10 @@ for env in bld.gen_python_environments():
 		realname='samba/dcerpc/messaging.so'
 		)
 
-if bld.PYTHON_BUILD_IS_ENABLED():
-    bld.SAMBA_SCRIPT('python_dcerpc_init',
-                 pattern='rpc/dcerpc.py',
-                 installdir='python/samba/dcerpc',
-                 installname='__init__.py')
+	if bld.PYTHON_BUILD_IS_ENABLED():
+		bld.SAMBA_SCRIPT('python_dcerpc_init',
+			pattern='rpc/dcerpc.py',
+			installdir='python/samba/dcerpc',
+			installname='__init__.py')
 
-    bld.INSTALL_FILES('${PYTHONARCHDIR}/samba/dcerpc', 'rpc/dcerpc.py', destname='__init__.py')
+	bld.INSTALL_FILES('${PYTHONARCHDIR}/samba/dcerpc', 'rpc/dcerpc.py', destname='__init__.py')
-- 
2.13.5

